test/quantization/core/test_quantized_module.py

```diff
@@ -155,15 +154,21 @@ def _test_linear_api_impl(self, batch_size, in_features, out_features, use_bias,
                          linear_unpack(loaded_qlinear._packed_params._packed_params))
         self.assertEqual(qlinear.scale, loaded_qlinear.scale)
         self.assertEqual(qlinear.zero_point, loaded_qlinear.zero_point)
-        # make sure loaded_qlinear has the same dir as qlinear since
-        # scripting the module will add __overloads__ to __dict__
         self.checkScriptable(loaded_qlinear, [[X_q]], check_save_load=True)
```

Comment:
actually should we do a copy before scripting so that it does not change load_qlinear?

Reply:
I know we just added copy in this PR but that feels like the correct thing to do..

Reply:
thanks, good point, updated the code to do that